src/hclib-deque.c

```diff
@@ -39,32 +39,38 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include "hclib-internal.h"
 #include "hclib-atomics.h"
 
-void deque_init(deque_t *deq, void *init_value) {
+#if 0 // UNUSED
```

Comment:
Actually, removing this should be fine since it's all limited to the deque. The other code I was talking about is marked with HC_CUDA/HCUPC preprocessor conditionals. I'll delete this and update the branch shortly.